omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_0/ConceptProposalResource2_0Test.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_0;
+
+import org.junit.Before;
+import org.openmrs.ConceptProposal;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.RestTestConstants1_8;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+import org.openmrs.util.OpenmrsConstants;
+
+public class ConceptProposalResource2_0Test extends BaseDelegatingResourceTest<ConceptProposalResource2_0, ConceptProposal> {
```

Comment:
As currently written, what is the value of this class?

Reply:
I've added more props assertions. https://github.com/openmrs/openmrs-module-webservices.rest/pull/494/commits/824349970e3a52a252dbd2483a6c2fb4bd3e3376